components/x-article-save-button/src/ArticleSaveButton.scss

```diff
@@ -0,0 +1,36 @@
+@import 'o-icons/main';
+@import 'o-colors/main';
+@import 'o-typography/main';
+
+$icon-size: 38px;
+
+@mixin background-icon($name, $color) {
+	@include oIconsGetIcon($name, oColorsGetPaletteColor($color), $icon-size, $iconset-version: 1);
+}
+
+.article-save-button {
+	display: inline-block;
+}
+
+.article-save-button__button {
```

Comment:
this isn't documented but because we're using CSS modules, BEM-style class names aren't necessary. i'd recommend using short, valid-JS-identifier classes for readability, e.g. `.articleSave`, `.button`, `.saved`, then you can reference them as `styles.articleSave` etc